x-pack/plugins/security_solution/server/endpoint/routes/artifacts/download_exception_list.ts

```diff
@@ -54,7 +54,11 @@ export function registerDownloadExceptionListRoute(
       // The ApiKey must be associated with an enrolled Fleet agent
       try {
         scopedSOClient = endpointContext.service.getScopedSavedObjectsClient(req);
-        await authenticateAgentWithAccessToken(scopedSOClient, req);
+        await authenticateAgentWithAccessToken(
```

Comment:
@paul-tavares FYI

with the feature flag OFF artifacts were still being downloaded